documentation-site/examples/notification-circle/offset.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from 'react';
+import {Tag, SIZE} from 'baseui/tag';
+import {NotificationCircle, COLOR, PLACEMENT} from 'baseui/badge';
+
+export default function Example() {
+  return (
+    <React.Fragment>
+      <NotificationCircle
+        content={11}
+        color={COLOR.positive}
+        placement={PLACEMENT.topRight}
+        horizontalOffset="0px"
+        verticalOffset="0px"
```

Comment:
is setting offsets to 0 necessary?

Reply:
The offsets override the default positions given the type of badge and its placement. In this case, the badge is being placed on a Tag, and the Tag component has margin, so the default positioning places the badge floating a few pixels away from the edge of the Tag. It just so happens that setting the offsets to `0px` lines up the badge nicely. But I do see how that's confusing -- perhaps it makes sense to choose a different element as an anchor.